DungeonGame.py

```
@@ -0,0 +1,122 @@
+
+        event = random.choice(["nothing", "enemy", "treasure"])
+        if event == "enemy":
+            encounter_enemy(player)
+        elif event == "treasure":
+            find_treasure(player)
+        else:
+            print("🔎 The area is quiet...")
+
+    if player.health > 0:
+        print("\n🏆 You found the exit! But a boss stands in your way...")
+        boss = Enemy("Dungeon Lord", 80, 20)
```

Ruff (0.8.2)

Local variable `boss` is assigned to but never used

Remove assignment to unused variable `boss`

(F841)